instrumentation/redisson-3.0/src/main/java/io/opentelemetry/instrumentation/auto/redisson/RedissonClientTracer.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.auto.redisson;
+
+import io.netty.channel.Channel;
+import io.opentelemetry.instrumentation.api.tracer.DatabaseClientTracer;
+import io.opentelemetry.instrumentation.auto.api.jdbc.DbSystem;
+import java.net.InetSocketAddress;
+import java.util.List;
+import org.redisson.client.RedisConnection;
+import org.redisson.client.protocol.CommandData;
+import org.redisson.client.protocol.CommandsData;
+
+public class RedissonClientTracer extends DatabaseClientTracer<RedisConnection, Object> {
+
+  public static final RedissonClientTracer TRACER = new RedissonClientTracer();
+
+  @Override
+  protected String getInstrumentationName() {
+    return "io.opentelemetry.auto.redisson-3.0";
+  }
+
+  @Override
+  protected String normalizeQuery(Object args) {
+    String commandName = "Redis Command";
+    // get command
+    if (args instanceof CommandsData) {
+      List<CommandData<?, ?>> commands = ((CommandsData) args).getCommands();
+      if (commands != null && !commands.isEmpty()) {
+        commandName = commands.get(0).getCommand().getName() + "... [bulk]";
```

Comment:
I don't think we do any truncation in other places, like Lettuce or DB instrumentation. It probably is a good idea but we should do it consistently for all the instrumentation, how about storing everything for now?

Reply:
Java SDK already supports truncating too large attributes. So this better be left to them :)